ui/components/ui/identicon/identicon.component.js

```diff
@@ -11,7 +13,7 @@ const getStyles = (diameter) => ({
   borderRadius: diameter / 2,
 });
 
-export default class Identicon extends PureComponent {
```

Comment:
React.PureComponent’s shouldComponentUpdate() only shallowly compares the objects. If these contain complex data structures, it may produce false-negatives for deeper differences. 

That is exactly our case, because we pass a complex object(s) in props and it keeps re-rendering. I was getting a lot of warnings from this tool: https://github.com/welldone-software/why-did-you-render